### ISSUE TITLE
Guest book ?

### ISSUE THREAD

**Issue by gdaudin** [medialab/toflit18#215]
I am not sure it would be widely used, but doing a guest book might help know our users better.

**Comment by paulgirard** [comment]
I don't think that's a good idea. It would most likely open the door to many spam messages to moderate for only a little (if any) user messages.
We might add a how-to submit feedback as github issues or an email (again => spam).

**Comment by gdaudin** [comment]
Fair enough. But I just listened to that https://medialab.sciencespo.fr/actu/transnum-en-video-un-tournant-numerique-pour-lhistoire-economique/ (remember it?) and I believe we should try to get information on our users.
